config/locales/devise/en.yml

```diff
@@ -115,6 +115,8 @@ en:
         That security code is invalid. You can try entering it again or request a new
         one-time security code.
       invalid_personal_key: That personal key is invalid.
+      max_generic_login_attempts_reached: >
+        Your account is temporarily locked.
```

Comment:
Hey @jeanninehunter can you take a look at this and let me know if the copy is OK? See PR description for context on when a user would see this message